src/app/containers/RadioPageBlocks/__snapshots__/index.test.jsx.snap

```diff
@@ -121,6 +121,7 @@ exports[`Radio Page Blocks should match snapshot for AMP 1`] = `
               layout="fill"
               sandbox="allow-scripts allow-same-origin"
               src="https://www.test.bbc.co.uk/ws/av-embeds/media/bbc_amharic_radio/liveradio/am/amp"
+              title="Audio  player"
```

Comment:
Looks like this snapshot has generated with an extra space between `audio` and `player`.